components/navigation.css

```
@@ -0,0 +1,23 @@
+.navigation {
+  display: flex;
+  justify-content: space-between;
+  align-content: center;
+  background-color: var(--color-white);
+  padding: 12px 29px;
+  border-radius: 10px;
+  box-shadow: 0 2px 11px var(--shadow-grey);
```

I think the shadow is medium blue with the opacity of 20% instead of grey, like this: 
```suggestion
--shadow-dim-blue: 0 2px 11px var(--color-dim-blue-op-20);
/* (--color-dim-blue-op-20: rgb(117, 137, 162, 0.2); */
```